### PR TITLE
Build dmg file only when running on Mac os

### PR DESCRIPTION
**Pull request by baptistemesta**
Since dmg file can only be built on Mac os, add a condition to do that only on ...Mac os :)

see https://github.com/unknown-horizons/godot-port/pull/53
https://en.wikipedia.org/wiki/Uname#Examples

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -41,17 +41,24 @@ fi
...
 # Builds.
+
+echo "Exporting Linux binary... "
 $GODOT --path . --export linux "Builds/Desktop/$PROJECTNAME.x86_64"
+echo "Exporting Windows binary... "
 $GODOT --path . --export windows "Builds/Desktop/$PROJECTNAME.exe"
-$GODOT --path . --export mac "Builds/Desktop/$PROJECTNAME.dmg"
+
+if [ "$(uname -s)" == "Darwin" ]; then
+    echo "Exporting Mac os binary... "
```
Comment (aaronfranke) [inline build.sh]: Can you change this to "MacOS" or "macOS"? I prefer the former, though I mentioned the latter since it's Apple's official branding.

**Review by aaronfranke — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (baptistemesta) [reply build.sh]: done

**Review by aaronfranke — APPROVED** [review]

**Comment by aaronfranke** [comment]
Squashing-and-merging, thanks!

**Comment by baptistemesta** [comment]
👍

**Pull request merged**